### ISSUE TITLE
OSError: [WinError 127] The specified procedure could not be found

### ISSUE THREAD

**Issue by MXA404** [pyg-team/pytorch_geometric#6730]
### 😵 Describe the installation problem

i was trying to work with a certain transformer for the point cloud but I faced some issues I tried to resolve most of the issues for libraries but then I faced this error.

lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: [WinError 127] The specified procedure could not be found

### Environment

* PyG version:
* PyTorch version: 1.13.1
* OS: windows 11
* Python version: 3.9.16
* CUDA/cuDNN version: 11.6.2
* How you installed PyTorch and PyG (`conda`, `pip`, source): conda
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by rusty1s** [comment]
Which PyG version are you using? By now, all these issues should be resolved when installing from master
```
pip install git+https://github.com/pyg-team/pytorch_geometric.git
```
Let me know if that works for you.

**Comment by htrieu93** [comment]
Just want to chime in to say that I got this exact error today when I tried to import torch-geometric. Weirdly I got it to work just fine the first time, but then after I un/reinstalled C++ this error popped up.
@rusty1s: I will try your solution on Monday and report back if it works for me
**Environment**
- PyG: 2.2.0
- PyTorch version: 1.7.0 (tried a bunch of other versions i.e: 1.8, 1.14, etc. but due to cpu only with 1.7.0 was I able to instable the complementary packages)
- OS: Window 10
- CUDA: CPU
- How you installed PyTorch and Py: pip
- Any other relevant information: can't check cuz at work

**Comment by MXA404** [comment]
i tried the solution from @rusty1s i think the issue are related to torch-scatter and torch-sparse some how and in the error it shows that 

UserWarning: An issue occurred while importing 'torch-scatter'. Disabling its usage. Stacktrace: [WinError 127] The specified procedure could not be found

UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: [WinError 127] The specified procedure could not be found
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. 

i will try to install them again and check if you any method that will be great also i will try it on another laptop to confirm the issue

**Comment by MXA404** [comment]
my pytorch is 1.13.1

**Comment by htrieu93** [comment]
@MXA404 do you have to use pytorch 1.13? Could you try pytorch + pyg 1.12? I had the exact same problem with 1.13 but 1.12 worked fine for me

**Comment by rusty1s** [comment]
> i tried the solution from @rusty1s i think the issue are related to torch-scatter and torch-sparse some how and in the error it shows that

Yeah, the warnings are expected here in case it fails to load the libraries. PyG should still work fine without these libraries.

**Comment by MXA404** [comment]
@htrieu93 i tried 1.12 and it fix the issue for the libraries I still got an issue of sharedarray I will install the library 

thank you so much I really appreciate the help
